rasa_sdk/knowledge_base/actions.py

```diff
@@ -45,7 +50,7 @@ def utter_attribute_value(
         object_name: Text,
         attribute_name: Text,
         attribute_value: Text,
-    ) -> None:
```

Comment:
should we remove these? as far as I remember the pep suggested to keep them, but not sure

Reply:
The only thing that I could find in PEP regarding `None` return values is for the `__init__()` method, where it's considered a best practice to return `None`, otherwise the type checker might not validate the types for the method.

The reason `mypy` complains is because of this: https://mypy.readthedocs.io/en/stable/error_code_list.html#check-that-called-function-returns-a-value-func-returns-value

I would agree that it's better to make a clear distinction between a function that is supposed to return something (and you're suppose to do something with the returned value), versus a function that doesn't return anything (and you shouldn't do anything with the return value).